packages/e2e/helpers/github.helper.ts

```diff
@@ -0,0 +1,22 @@
+import axios, { AxiosInstance } from "axios"
+
+class GitHubHelper {
+  private gitHubInstance: AxiosInstance
+
+  constructor() {
+    this.gitHubInstance = axios.create({
+      baseURL: "https://api.github.com/repos/mudita/MuditaOS",
+      auth: {
+        username: process.env.TEST_GITHUB_USER,
+        password: process.env.TEST_GITHUB_PASSWORD,
+      },
```

Comment:
🟡
I suggest using token

Tokens offer [a number of security benefits](https://github.blog/2020-07-30-token-authentication-requirements-for-api-and-git-operations/#background) over password-based authentication:

Unique – tokens are specific to GitHub and can be generated per use or per device
Revocable – tokens can can be individually revoked at any time without needing to update unaffected credentials
Limited – tokens can be narrowly scoped to allow only the access necessary for the use case
Random – tokens are not subject to the types of dictionary or brute force attempts that simpler passwords that you need to remember or enter regularly might be

Reply:
https://github.com/mudita/mudita-center/pull/611/commits/b7e381a9b47bf9ffbf1f014e9fe4df4194fa8563